api/apps/api/src/modules/scenarios/scenarios-crud.service.ts

```diff
@@ -132,6 +136,16 @@ export class ScenariosCrudService extends AppBaseService<
     return model;
   }
 
+  async assignCreatorRole(scenarioId: string, userId: string): Promise<void> {
+    await this.userScenarios.save(
+      this.userScenarios.create({
+        scenarioId,
+        userId,
+        roleName: Roles.scenario_owner,
+      }),
+    );
+  }
+
```

Comment:
as ACL is not standalone module, `ScenariosModule` should rather call it directly instead managing privileges on its own.